version.sbt

```diff
@@ -1 +1 @@
-ThisBuild / version := "0.18.0-SNAPSHOT"
+ThisBuild / version := "0.18.0"
```

Comment:
on snapshot publish, this will get prepended with SNAPSHOT by the CI. This is also how we can differentiate snapshot vs non-snapshot release on build.sbt file.

Reply:
I wonder if we can use:

https://github.com/sbt/sbt-dynver

Reply:
sounds fine to me will implement I like the "fetch from recent tag" idea.